Makefile

```diff
@@ -18,6 +18,25 @@ build-z3:
 build:
 	stack build --copy-bins --fast -j $(shell nproc)
 
+build-cache:
```

Comment:
hmm can we still do this with a master `stack.yaml` file?

Reply:
you can type stack build... however I think the master `stack.yaml` flag has some extra flags(?) it seems to ruin my cache at least for some reason.